src/diff/index.js

```diff
@@ -223,11 +215,8 @@ export function diff(
 
 		if ((tmp = options.diffed)) tmp(newVNode);
 	} catch (e) {
-		if (isHydrating) {
-			// Before bailing out, mark the current VNode with the DOM element and hydration state.
-			// We can use this information if we return here to render later on.
-			oldVNode._hydrateDom = newVNode._dom = oldDom;
-		}
+		newVNode._dom = oldDom; // is this needed?
```

Comment:
The Suspense tests pass if we still keep `isHydrating` check

Reply:
IIRC `newVNode._dom = oldDom;` is needed because the diff remove the DOM otherwise it removes the existing DOM in next render.

Reply:
I'll try to get this fixed.